lighthouse-extension/gulpfile.js

```diff
@@ -141,6 +141,11 @@ gulp.task('browserify-lighthouse', () => {
         bundle = bundle.require(artifact, {expose: artifact.replace(corePath, './')});
       });
 
+      // browerify's url shim doesn't work with .URL in node_modules, so expose our own
+      // @see https://github.com/GoogleChrome/lighthouse/issues/5273
+      const pathToURLShim = require.resolve('../lighthouse-core/lib/url-shim.js');
+      bundle = bundle.require(pathToURLShim, {expose: 'url'});
```

Comment:
where is this lowercase usage of it used? i'm not following what the fix is here.

Reply:
inside robots-parser it does `var URL = require('url').URL`